src/utils/records.ts

```diff
@@ -272,14 +272,54 @@ export class Records {
     return filterCopy;
   }
 
+
+  public static tagsFilterIsPrefix(filter: RecordsWriteTagsFilter): filter is PrefixFilter {
+    return typeof filter === 'object' && ('prefix' in filter && typeof filter.prefix === 'string');
+  }
+
+  /**
+   * This will create individual keys for each of the tags that look like `tag.tag_property`
+   */
+  public static flattenTags(tags: RecordsWriteTags): KeyValues {
+    const tagValues:KeyValues = {};
+    for (const property in tags) {
+      const value = tags[property];
+      tagValues[`tag.${property}`] = value;
+    }
+    return tagValues;
+  }
+
+  /**
+   * This will create individual keys for each of the tag filters that look like `tag.tag_filter_property`
+   */
+  public static flattenTagFilters( tags: { [property: string]: RecordsWriteTagsFilter}): Filter {
```

Comment:
Change this to `private` and optionally rename to something like `convertTagsFilter()` to align with the naming of `convertFilter` and `RecordsWriteTagsFilter`.